src/UI/Buyer/src/app/shared/containers/register/register.component.ts

```diff
@@ -78,10 +80,19 @@ export class RegisterComponent implements OnInit, OnDestroy {
   private setForm() {
     const formObj = {
       Username: ['', Validators.required],
-      FirstName: ['', Validators.required],
-      LastName: ['', Validators.required],
-      Email: ['', [Validators.required, Validators.email]],
-      Phone: [''],
+      FirstName: [
+        '',
+        [Validators.required, Validators.pattern(this.regexService.HumanName)],
+      ],
+      LastName: [
+        '',
+        [Validators.required, Validators.pattern(this.regexService.HumanName)],
+      ],
+      Email: [
+        '',
+        [Validators.required, Validators.pattern(this.regexService.Email)],
```

Comment:
Isn't there already a Validators.email we could use?

Reply:
Oh yeah, I guess I was on a role with regex and forgot about that. Yeah, lets use Validators.email.